api/package.json

```diff
@@ -18,7 +18,7 @@
     "license": "MIT",
     "devDependencies": {
         "@types/node": "^16.0.0",
-        "@types/vscode": "^1.64.0"
+        "@types/vscode": "^1.90.0"
```

Comment:
We should pin to a specific version so that `vsce` doesn't get mad; i.e. remove the `^`. You will also need to `npm i` again as it actually picked up `@types/vscode@1.103.0` in the lockfile.

Reply:
Ohh it'll see that we're using proposed api that isn't in that types version?

Reply:
`vsce` will complain if you use `@types/vscode` higher than the minimum engine version specified in the extension's package.json.

Since this is a package itself (not an extension), the `@types/vscode` dev dependency won't bubble up to the extension, and `vsce` won't actually notice--but the same problems can result at runtime.